### DISCUSSION TITLE
Custom "Edit Javascript" and "Edit HTML" options inside the dashboard portlet

### DISCUSSION THREAD

**Discussion by JatinVenkatesh** [apache/superset#28332] [Ideas]
We currently possess the capability to customize dashboards through the 'Edit CSS' feature. It would significantly enhance the flexibility and extensibility of Apache Superset if we could incorporate additional functionalities such as 'Edit JavaScript' and 'Edit HTML' directly within the dashboard editing interface. This expansion would empower users to tailor their dashboards more comprehensively to meet their specific requirements.

**Comment by kayuzee** [comment]
Totally agree, although in the meantime you can leverage the tailsjs file https://stackoverflow.com/questions/76355863/how-to-write-custom-javascript-in-superset-which-is-installed-in-docke
